SPECS/python-poetry-core/python-poetry-core.spec

```diff
@@ -0,0 +1,130 @@
+%global pypi_name poetry-core
+
+Summary:        Poetry PEP 517 Build Backend
+Name:           python-poetry-core
+Version:        1.0.7
+Release:        2%{?dist}
+
+# We bundle a lot of libraries with poetry, which itself is under MIT license.
+# Here is the list of the libraries with corresponding licenses:
+
+# attrs: MIT
+# jsonschema: MIT
+# lark-parser: MIT
+# packaging: ASL 2.0 or BSD
+# pyparsing: MIT
+# pyrsistent: MIT
+# six: MIT
+# tomlkit: MIT
+
+License:        MIT and (ASL 2.0 or BSD)
+URL:            https://github.com/python-poetry/poetry-core
+Source0:        %{url}/archive/%{version}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  pyproject-rpm-macros
+BuildRequires:  python3-devel
+%if %{with_check}
+BuildRequires:  gcc
+BuildRequires:  git-core
+BuildRequires:  python3-pip
+BuildRequires:  python3-pytest
+%endif
+Requires:       python3
+
+%global _description %{expand:
+A PEP 517 build backend implementation developed for Poetry.
+This project is intended to be a light weight, fully compliant, self-contained
+package allowing PEP 517 compatible build frontends to build Poetry managed
+projects.}
+
+%description %_description
+
+%package -n python3-poetry-core
+Summary:        %{summary}
+
+# Previous versions of poetry included poetry-core in it
+Conflicts:      python%{python3_version}dist(poetry) < 1.1
+# The bundled versions are taken from poetry/core/_vendor/vendor.txt
+Provides:       bundled(python3dist(attrs)) = 20.3
+Provides:       bundled(python3dist(jsonschema)) = 3.2
+Provides:       bundled(python3dist(lark-parser)) = 0.9
+Provides:       bundled(python3dist(packaging)) = 20.9
+Provides:       bundled(python3dist(pyparsing)) = 2.4.7
+Provides:       bundled(python3dist(pyrsistent)) = 0.16.1
+Provides:       bundled(python3dist(six)) = 1.15
+Provides:       bundled(python3dist(tomlkit)) = 0.7
```

Comment:
Would prefer to not see all of this vendoring- we try to avoid it, as a policy, whenever possible. Looks like [the issue which caused Fedora to re-vendor](https://github.com/python-poetry/poetry/issues/4264) is both fixed and non-applicable to Mariner (since we don't ship Poetry at the moment).

We should take up [Fedora's de-vendoring patch](https://src.fedoraproject.org/rpms/python-poetry-core/raw/rawhide/f/poetry-core-1.0.2-devendor.patch), with any necessary updates made.